setup.cfg

```diff
@@ -1,3 +1,3 @@
 [flake8]
-max-line-length = 88
-ignore = E203, W503, E501, E231  # Black
+max-line-length = 2424
+ignore = E203, E402, W503
```

Comment:
**Issue**: We did not need E402 before, why now?

Reply:
Mods to `docs/conf.py`

Reply:
I would prefer not to add ignore rules affecting the whole code base for an issue in `docs/conf.py`.  If there is an issue in that file, it can be marked using `# noqa`.